Expensify/App
# Add an option to global create menu for “New Workspace”

### Details

As the title says.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/166033

### Tests
1. Open Expensify.cash
2. Make sure you're in the `freePlan` beta.
3. Click this button:
![image](https://user-images.githubusercontent.com/6759612/122246985-4d8f9000-cec7-11eb-9bcb-36209f9edc70.png)
4. Make sure the "New Workspace" item is there:
![image](https://user-images.githubusercontent.com/6759612/122247079-6304ba00-cec7-11eb-91ec-668013be3e3b.png)

### QA Steps
1. Open Expensify.cash
2. Make sure you're in the `freePlan` beta.
3. Click this button:
![image](https://user-images.githubusercontent.com/6759612/122246985-4d8f9000-cec7-11eb-9bcb-36209f9edc70.png)
4. Make sure the "New Workspace" item is there:
![image](https://user-images.githubusercontent.com/6759612/122247079-6304ba00-cec7-11eb-91ec-668013be3e3b.png)

You should also follow the same steps with an account that is not in the `freePlan` beta to make sure the item is not in the list.

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/6759612/122249061-d3f8a180-cec8-11eb-91e8-c459501ecd1d.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/6759612/122250091-aa8c4580-cec9-11eb-9d1f-53714a25a582.png)

#### Desktop
![image](https://user-images.githubusercontent.com/6759612/122254226-3653a100-cecd-11eb-8a12-7eb121c029b5.png)

#### iOS
![image](https://user-images.githubusercontent.com/6759612/122256355-466c8000-cecf-11eb-8e84-8b9e3c63e636.png)

#### Android
![image](https://user-images.githubusercontent.com/6759612/122255325-4cae2c80-cece-11eb-8ec5-37106604237f.png)
